src/admin-views/all-tickets/filters/status-select.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * All Tickets list table status filter template.
+ *
+ * @since TBD
+ *
+ * @var \Tribe__Template  $this            Current template object.
+ * @var string[]          $status_options  The list table for the All Tickets screen.
+ * @var string            $current_status  Currently selected filter.
+ */
+
+?>
+<select name="status-filter" id="tec-tickets-all-tickets-status-filter">
+	<?php foreach ( $status_options as $value => $label ) : ?>
```

Comment:
Is $status_options reflecting statuses that are present in the DB ? Or are they simply all the values ? If it's the first case, probably we should check that its `count() > 1` before we print the `<select/>`

Reply:
Do we also need an `All` option ?

Reply:
`status` is a little confusing in this case. They are not WP statuses. We have a set list of them. They are listed [here](https://github.com/the-events-calendar/event-tickets/blob/d11100b18c16d90a33bf1cce49362f9d88947ef5/src/Tickets/Admin/All_Tickets/List_Table.php#L901-L907). `All` is included as an option.